benchmarks/ClusterRewards.ts

```diff
@@ -1,8 +1,11 @@
-import { ethers, waffle } from "hardhat";
+import { ethers } from "hardhat";
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers'
 import { benchmark as benchmarkDeployment } from "./helpers/deployment";
 import { initDataFixture } from "./fixtures/ClusterRewards";
 import { BigNumber, BigNumberish, constants, Contract, PopulatedTransaction, Signer, utils } from "ethers";
 import { randomlyDivideInXPieces, skipTime } from "./helpers/util";
+import { ClusterRewards, ClusterRewards__factory } from "../typechain-types";
+import { IArbGasInfo__factory } from "../typechain-types/factories/contracts/staking/ClusterSelector.sol";
```

Comment:
can't IArbGasInfo__factory be imported from "../typechain-types" as well?

Reply:
Import not used, hence removed now.